pkg/build/builder/dockerutil.go

```diff
@@ -39,14 +44,18 @@ func pushImage(client DockerClient, name string, authConfig docker.AuthConfigura
 		opts.OutputStream = os.Stderr
 	}
 	var err error
+
 	for retries := 0; retries <= DefaultPushRetryCount; retries++ {
-		err = client.PushImage(opts, authConfig)
+		err := client.PushImage(opts, authConfig)
```

Comment:
hmm, weird that the compiled didn't shout. Must leave it there when I was re-working the logic.